src/main/java/org/stellar/sdk/responses/sorobanrpc/SendTransactionResponse.java

```diff
@@ -0,0 +1,38 @@
+package org.stellar.sdk.responses.sorobanrpc;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@AllArgsConstructor
+@Value
+public class SendTransactionResponse {
+  @SerializedName("status")
```

Comment:
can remove the @SerializedName annotations when it matches the java class member name, gson will by default serialize to that name.

Reply:
Fixed.